CourseManager/Components/CourseDetailsRowView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  CourseDetailsRowView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-28.
+//
+
+import SwiftUI
+
+struct CourseDetailsRowView: View {
+    var course = CourseInformation(name: "SwiftUI", url: "swiftUI.com", imageName: "Build SwiftUI App", progressPercentage: 0.5, gitHubURL: "", status: .complete, dateOfCompletion: Date(), comments: "great course, will be implenting")
+    var body: some View {
+        VStack {
+            VStack(alignment: .leading, spacing: 10) {
+                Text("Course Name")
+                    .font(.headline)
+                    .fontWeight(.bold)
+                Text(course.name)
+                    .foregroundColor(.secondary)
+                Divider()
+                Text("Course Link")
+                    .font(.headline)
+                    .fontWeight(.bold)
+                if let url = URL(string: course.url) {
+                    Link(course.url, destination: url)
+                }
+                Divider()
+                Text("Status")
+                    .font(.headline)
+                    .fontWeight(.bold)
+                Text(course.status.localizedName)
+                    .foregroundColor(.secondary)
+                Divider()
+            }
+            VStack(alignment: .leading, spacing: 10) {
+                if course.status == .complete {
```

Comment:
`if` here looks like, that it should be merged into Text

```
Text(course.status == .complete ? "Date completed" : "Comments")
                     .font(.headline)
                     .fontWeight(.bold)
```

Reply:
Hi Yura, I actually wanted to omit the whole row if course.state != .complete. Comments stay in all cases. I tried to use ternary operator in this situation but I don't think it's possible.

Reply:
Hi, yes you are right, I wrote something strange :). Maybe it worth to move this out to separate view

````
Text("Comments")
                     .font(.headline)
                     .fontWeight(.bold)
                 Text(course.comments)
                     .foregroundColor(.secondary)
```

Reply:
You're right, it's a bit of a mess. 
I've tidied up the whole VStack: 

   VStack(alignment: .leading, spacing: 10) {
            courseNameRow
            Divider()
            courseLinkRow
            Divider()
            statusRow
            Divider()
            dateCompleteRow
            Divider()
            commentsRow
        }
        .frame(maxWidth: .infinity, alignment: .leading)
        .padding()